smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -377,7 +377,7 @@ def devicesList(selector = '') {
 			state.lifxAccessToken = null
 		} else {
 			log.debug("No response from device list call. ${resp.status} ${resp.data}")
-			return []
+			return null
```

Comment:
instead of returning null and failing later with NPE, can we throw an exception here with the right message? I am worried that this might result in a few thousand NPE on the dashboard and might send us down another rabbit hole